templates/macros/helpers.html

```diff
@@ -27,3 +27,11 @@
   } %}
 <a href="mailto:{{email}}{% if subject %}?subject={{ subject|urlencode }}{% endif %}" {{attr|xmlattr}}>{{email}}</a>
 {%- endmacro -%}
+
+{%- macro interviewer_note(title='', type='') -%}
+  {% if type == 'Interstitial' %}
+    <h1 class="question__title"><mark class="instruction">{{_("Interviewer Note:")}}</mark> {{ title }}</h1>
```

Comment:
I feel like "Interviewer Note" text is something that should really be defined and not only in the templates. May be worth seeing if any of the rest of the team have an opinion.

Reply:
I don't see an issue with it being in the templates, it's done like that in plenty of other places.